src/components/button-link/index.tsx

```diff
@@ -58,7 +58,7 @@ const ButtonLink = ({
 				rel={openInNewTab ? 'noreferrer noopener' : undefined}
 				target={openInNewTab ? '_blank' : '_self'}
 				onClick={onClick}
-				data-heap-track={dataHeapTrack}
+				data-heap-track={`button-link ${dataHeapTrack}`}
```

Comment:
💭 Maybe `dataHeapTrack` should be defaulted to `''` in the props destructuring for the cases where it's not passed in?

Reply:
ah true! one sec...

Reply:
✅ should be fixed now!